deploy.sh

```diff
@@ -157,12 +157,22 @@ svn checkout $SVNURL $SVNPATH --depth immediates
 svn update --quiet $SVNPATH/trunk --set-depth infinity
 
 echo "Ignoring GitHub specific files"
-svn propset svn:ignore "README.md
+# Use local .plugins_svnignore if present
+if [ -f ".plugins_svnignore" ]; then
+	echo "Using local .plugins_svnignore"
+	SVNIGNORE=$( <.pluginsvnignore )
```

Comment:
This file name doesn't seem consistent (missing underscore) with everything else.

Reply:
Must be a typo on my part when I changed from `.svnignore` to `.plugins_svnignore`

Reply:
I probably should just go with my first choice `.svnignore`

Reply:
Changed back to `.svnignore`